repos/dir.targets

```diff
@@ -166,7 +166,7 @@
   </Target>
 
   <Target Name="WriteVersions"
-          AfterTargets="Build;CopyPackage"
+          AfterTargets="CopyPackage"
```

Comment:
Good catch, I think my mistake was treating `AfterTargets` as "after all finish", but it's "after any finish".